e2e/alphabet-card-detail.test.ts

```diff
@@ -12,15 +12,19 @@ const multiTap = async (tappable: ITappableAsync, numberOfTaps: number): Promise
     }
 };
 
+require('dotenv').config({ path: '.production.env' });
```

Comment:
You either need to resolve the path to the `.env` (E.g. {staging|production|testing}.env) from an actual environment variable (set before running the build), or you need to be able to swap out the `.env` file (don't name it production.env in this case) for each given environment (conceptually)
> > > rm app.env && staging.env app.env %% npm run build

Reply:
This line is now removed.